### PR TITLE
Footer gayle

### PR DESCRIPTION
**Pull request by Gayle-Thompson-Igwebike**

### CODE REVIEW

File: src/Footer.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "No4 Women Avenue, London.",
+    "E1 4UD",
+    "hello@bwcma.com",
+    "123-456-7890",
+  ];
+  const [Name, Address, PostCode, Email, Tel] = FooterArray;
+  return (
+    <div className="footer-container">
+      <div className="footer-details">
+        <ul className="footer-address">
```
[the remark this answers is not in the document]
Comment (Gayle-Thompson-Igwebike) [reply src/Footer.js]: Now update Karleen. Thanks

**Review by Gayle-Thompson-Igwebike — COMMENTED** [review]
Updated mobile version

File: src/Footer.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      
+      {/* <div id="operating-hours">
+        <OperatingHours />
+      </div> */}
+
+      <div id="contactus-form">
+        <ContactUsForm />
+      </div>
+
+      <div id="socialmedia-icons">
```
[the remark this answers is not in the document]
Comment (Gayle-Thompson-Igwebike) [reply src/Footer.js:46]: Text area now resolved.

File: src/Footer.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+          <div id="submit-btn">
+            <SubmitButton />
+          </div>
+        </div>
+      </div>
+
+      <div id="socialmedia-icons">
+        <SocialMediaIcons />
+      </div>
```
Comment (karleenmsrichards) [inline src/Footer.js:48]: Please make the social media icons black and place them under the telephone contact.

Comment (Gayle-Thompson-Igwebike) [reply src/Footer.js]: Updated as requested. Thanks Karleen

**Review by Gayle-Thompson-Igwebike — COMMENTED** [review]
All requests for icons and contact form in mobile screen now updated.

**Review by karleenmsrichards — APPROVED** [review]
Reviewed, gr8 job ☺️

**Pull request merged**